cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,41 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 2.6.2 <http://docs.cask.co/cdap/2.6.2/index.html>`__
+=============================================================
+
+New Features
+------------
+
+- Added log rotation for CDAP programs running in YARN containers
+  (`CDAP-1295 <https://issues.cask.co/browse/CDAP-1295>`__)
+
+- Added the ability to submit to non-default YARN queues to provide 
+  :ref:`resource guarantees <resource-guarantees>`
+  for CDAP Master Services, CDAP Programs, and Explore Queries
+  (`CDAP-1417 <https://issues.cask.co/browse/CDAP-1417>`__)
+
+- Added the ability to :ref:`prune invalid transactions <tx-maintenance>`
+  (`CDAP-1540 <https://issues.cask.co/browse/CDAP-1540>`__)
+
+- Added the ability to specify 
+  :ref:`custom logback file for CDAP programs <application-logback>`
+  (`CDAP-1741 <https://issues.cask.co/browse/CDAP-1741>`__)
+
+
+.. CDAP Bug Fixes
+.. --------------
+
+
+.. _known-issues-262:
+
+Known Issues
+------------
+- See also the *Known Issues* of `version 2.6.1. <#known-issues-261>`_
+
+- CDAP works only with `node.js <http://nodejs.org>`__ versions 0.8.16 through 0.10.36.
```

Comment:
One more known issue (please rephrase): When the CLI starts up, it auto-connects to `localhost`. After a `connect <hostname>` command, all operations work except for Explore queries (i.e. `execute 'query'`) since the `ExploreClient` doesn't pick up the new change of hostname. The workaround is to start up the CLI with the `CDAP_HOSTNAME` environment variable set, so that the CLI autoconnects to that host on startup. There is no JIRA for this since it has already been fixed in 2.8 as part of a refactor of the CLI.